\chapter{Protein Structure and Function}

\section{The Shape and Structure of proteins}

Understanding protein
structure at the atomic level will allow us to describe how the precise
shape of each protein determines its function in the cell.

\subsection{The Shape of a protein is Specified by its amino acid Sequence}

Proteins are assembled from a set of 20 different amino acids, each with different chemical properties. A protein
molecule is made from a long chain of these amino acids, each linked
to its neighbor through a covalent peptide bond. Proteins
are therefore referred to as polypeptides or polypeptide chains. In each
type of protein, the amino acids are present in a unique order, called the
amino acid sequence, which is exactly the same from one molecule
of that protein to the next.

Each polypeptide chain consists of a backbone that supports the different 
amino acid side chains. The polypeptide backbone is made from
the repeating sequence of the core atoms of the amino acids that form
the chain. Projecting from this repetitive backbone are any of the 20 different 
amino acid side chains - the parts of the amino acids that are not
involved in forming the peptide bond.

Long polypeptide chains are very flexible: many of the covalent bonds
that link carbon atoms in an extended chain of amino acids allow free
rotation of the atoms they join. Thus proteins can in principle fold in an
enormous number of ways. Each folded chain is constrained by many dif-
ferent sets of weak noncovalent bonds that form within proteins. These
bonds involve atoms in the polypeptide backbone as well as atoms in
the amino acid side chains. The noncovalent bonds that help proteins
maintain their shape include hydrogen bonds, electrostatic attractions,
and van der Waals attractions.

An important factor governing the folding of any protein is the distribution 
of its polar and nonpolar amino acids.

\subsection{Proteins fold into a conformation of lowest energy}

Each type of protein has a particular three-dimensional structure, which
is determined by the order of the amino acids in its chain. The final folded
structure, or conformation, adopted by any polypeptide chain is determined 
by energetic considerations: a protein generally folds into the
shape in which the free energy (G) is minimized. Protein folding 
has been studied in the laboratory using highly purified proteins. A
protein can be unfolded, or denatured, by treatment with solvents that
disrupt the noncovalent interactions holding the folded chain together.
This treatment converts the protein into a flexible polypeptide chain that
has lost its natural shape. When the denaturing solvent is removed, the
protein often refolds spontaneously, or renatures, into its original conformation.
All the information necessary to specify the three-dimensional shape of a protein 
is cointained in its amino acid sequence.

Each protein normally folds into a single stable conformation. This conformation, 
however, often changes slightly when the protein interacts
with other molecules in the cell. This change in shape is crucial to the
function of the protein.

When proteins fold incorrectly, they sometimes form aggregates that can
damage cells and even whole tissues.

Although a protein chain can fold into its correct conformation without
outside help, protein folding in a living cell is generally assisted by special
proteins called molecular chaperones. These proteins bind to partly folded
chains and help them to fold along the most energetically favorable pathway.

\subsection{Proteins come in a wide Variety of complicated Shapes}

Proteins can be globular or fibrous; they can form filaments, sheets,
rings, or spheres.

Resolving a protein’s structure often begins with determining its amino
acid sequence, a task that can be accomplished in several ways. For
many years, protein sequencing was accomplished by directly analyzing 
the amino acids in the purified protein; the first protein sequenced
was the hormone insulin, in 1955. Today we can determine the order of
amino acids in a protein much more easily by sequencing the gene that
encodes it. Once the order of the nucleotides
in the DNA that encodes a protein is known, this information can be
converted into an amino acid sequence by applying the genetic code.

\subsection{The $\alpha$-helix and the $\beta$ sheet are common folding patterns}

When the three-dimensional structures of many different protein molecules 
are compared, it becomes clear that, although the overall
conformation of each protein is unique, two regular folding patterns are
often present. Both were discovered more than 50 years ago from studies 
of hair and silk. The first folding pattern to be discovered, called the
$\alpha$-helix, was found in the protein $\alpha$-keratin, which is abundant in skin and
its derivatives - such as hair, nails, and horns. Within a year of the discovery 
of the a helix, a second folded structure, called a $\beta$ sheet, was found
in the protein fibroin, the major constituent of silk.

These two folding patterns are particularly common because they result
from hydrogen bonds that form between the $N-H$ and $C=O$ groups in
the polypeptide backbone. Because the amino acid side chains are not
involved in forming these hydrogen bonds, $\alpha$ helices and $\beta$ sheets can
be generated by many different amino acid sequences.

\subsection{Helices Form Readily in Biological Structures}

A helix is a regular structure that resembles a spiral staircase.
Depending on the twist of the staircase, a helix is said to be either 
right-handed or left-handed.

An $\alpha$-helix is generated when a single polypeptide chain turns around
itself to form a structurally rigid cylinder. A hydrogen bond is made
between every fourth amino acid, linking the $C=O$ of one peptide bond to
the $N-H$ of another. This gives rise to a regular helix
with a complete turn every 3.6 amino acids.
Short regions of $\alpha$-helix are especially abundant in the proteins located
in cell membranes, such as transport proteins and receptors.
The polypeptide backbone, which is hydrophilic, is hydrogen-bonded to itself in 
the $\alpha$-helix, and it is shielded
from the hydrophobic lipid environment of the membrane by its protruding nonpolar side chains.

Sometimes two (or three) $\alpha$ helices will wrap around one another to
form a particularly stable structure known as a coiled-coil. This structure 
forms when the a helices have most of their nonpolar (hydrophobic)
side chains on one side, so that they can twist around each other with
these side chains facing inward—minimizing their contact with the aqueous cytosol.

\subsection{$\beta$ Sheets form rigid Structures at the core of Many proteins}

$\beta$ sheets are made when hydrogen bonds form between segments of polypeptide 
chains lying side by side. When the structure consists of neighboring polypeptide chains that
run in the same orientation (say, from the N-terminus to the C-terminus),
it is considered a parallel $\beta$ sheet; when it forms from a polypeptide chain
that folds back and forth upon itself - with each section of the chain running 
in the direction opposite to that of its immediate neighbors - the
structure is an antiparallel $\beta$ sheet. Both types of $\beta$ sheet
produce a very rigid, pleated structure, and they form the core of many
proteins.

\subsection{Proteins have Several levels of organization}

A protein’s structure begins with its
amino acid sequence, which is thus considered its primary structure. The
next level of organization includes the a helices and b sheets that form
within certain segments of a polypeptide chain; these folds are elements
of the protein’s secondary structure. The full, three-dimensional conformation 
formed by an entire polypeptide chain - including the $\alpha$ helices,
$\beta$ sheets, random coils, and any other loops and folds that form between
the N- and C-termini - is sometimes referred to as the tertiary structure. 
Finally, if a particular protein molecule is formed as a complex of more than one polypeptide
chain, then the complete structure is designated its quaternary structure.
Studies of the conformation, function, and evolution of proteins have
also revealed the importance of a level of organization distinct from
those just described. This organizational unit is the protein domain,
which is defined as any segment of a polypeptide chain that can fold
independently into a compact, stable structure.
The different domains of a protein are ofter associated with different functions.

\subsection{Few of the many possible polypeptide chains will be useful}

For a polypeptide that is $n$ amino acids long, $20^n$ different chains are possible.
However, only a very small fraction of this unimaginably large number
of polypeptide chains would fold into a stable, well-defined three-dimensional conformation.

\subsection{Proteins can be classified into families}

Once a protein had evolved a stable conformation with useful properties, 
its structure could be modified over time to enable it to perform
new functions. We know that this occurred quite often during evolution,
because many present-day proteins can be grouped into protein families, 
in which each family member has an amino acid sequence and a
three-dimensional conformation that closely resembles that of the other
family members.

\subsection{Large protein Molecules often contain More Than one polypeptide chain}

The same weak noncovalent bonds that enable a polypeptide chain to
fold into a specific conformation also allow proteins to bind to each other
to produce larger structures in the cell. Any region on a protein’s surface
that interacts with another molecule through sets of noncovalent bonds
is termed a binding site.
If a binding site recognizes the surface
of a second protein, the tight binding of two folded polypeptide chains
at this site will create a larger protein whose quaternary structure has a
precisely defined geometry. Each polypeptide chain in such a protein is
called a subunit.

In the simplest case, two identical folded polypeptide chains form a symmetrical
complex of two protein subunits (called a dimer) that is held
together by interactions between two identical binding sites.
Other proteins contain two or more different types of polypeptide chains.

\subsection{Proteins can assemble into Filaments, Sheets, or Spheres}

A chain of identical protein molecules can be formed if the
binding site on one protein molecule is complementary to another region
on the surface of another protein molecule of the same type. Because
each protein molecule is bound to its neighbor in an identical way, the
molecules will often be arranged in a helix that can be extended indefinitely. 
This type of arrangement can produce an extended protein filament.
Other sets of proteins associate to form extended sheets or tubes, 
as in the microtubules of the cell cytoskeleton, or cagelike spherical shells,
as in the protein coats of virus particles.

Many large structures, such as viruses and ribosomes, are built from a
mixture of one or more types of protein plus RNA or DNA molecules.

\subsection{Some Types of proteins have elongated fibrous Shapes}

Most of the proteins we have discussed so far are globular proteins, in
which the polypeptide chain folds up into a compact shape like a ball
with an irregular surface. Enzymes tend to be globular proteins: even
though many are large and complicated, with multiple subunits, most
have a quaternary structure with an overall rounded shape. 
In contrast, other proteins have roles in the cell that require them to
span a large distance. These proteins generally have a relatively simple,
elongated three-dimensional structure and are commonly referred to as
fibrous proteins.
An $\alpha$-keratin molecule is a dimer of two identical subunits, with the
long a helices of each subunit forming a coiled-coil.
These coiled-coil regions are capped at either end by globular domains
containing binding sites that allow them to assemble into ropelike inter-
mediate filaments - a component of the cell cytoskeleton that creates a
structural scaffold for the cell’s interior.
Fibrous proteins are especially abundant outside the cell, where they
form the gel-like extracellular matrix that helps cells bind together to form
tissues.

\subsection{Extracellular proteins are often Stabilized by covalent cross-linkages}

Many protein molecules are either attached to the outside of a cell’s
plasma membrane or secreted as part of the extracellular matrix. All such
proteins are directly exposed to extracellular conditions. To help maintain 
their structures, the polypeptide chains in such proteins are often
stabilized by covalent cross-linkages.
The most common covalent cross-links
in proteins are sulfur–sulfur bonds. These disulfide bonds (also called
S–S bonds) form as proteins are being exported from cells.
Disulfide bonds do not change the
conformation of a protein, but instead act as a sort of “atomic staple”
to reinforce its most favored conformation.

\section{How proteins work}

\subsection{All proteins bind to other molecules}

All proteins stick, or bind, to other molecules. In some cases this binding is
very tight; in others it is weak and short-lived.
Regardless of its strength, the binding
of proteins to other biological molecules always shows great specificity:
each protein molecule can bind to just one or a few molecules out of
the many thousands of different molecules it encounters. Any substance
that is bound by a protein - whether it is an ion, a small molecule, or a
macromolecule - is referred to as a ligand for that protein.

When molecules have poorly matching surfaces, few noncovalent bonds
are formed and the two molecules dissociate as rapidly as they come
together.

The region of a protein that associates with a ligand, known as its binding site, 
usually consists of a cavity in the protein surface formed by a
particular arrangement of amino acids. Other regions on the surface often provide 
binding sites for different ligands, allowing the protein’s
activity to be regulated, as we shall see later. Yet other parts of the protein
may be required to attract or attach the protein to a particular location in
the cell - for example, the hydrophobic $\alpha$-helix of a membrane-spanning
protein allows it to be inserted into the lipid bilayer of a cell membrane.

\subsection{The Binding Sites of antibodies are especially Versatile}

Antibodies, or immunoglobulins, are proteins produced by the immune
system in response to foreign molecules, such as those on the surface
of an invading microorganism. Each antibody binds to a particular target 
molecule extremely tightly, either inactivating the target directly or
marking it for destruction. An antibody recognizes its target (called an
antigen) with remarkable specificity, and because there are potentially
billions of different antigens that a person might encounter, we have to
be able to produce billions of different antibodies.

Antibodies are Y-shaped molecules with two identical binding sites that
are each complementary to a small portion of the surface of the antigen
molecule.

\subsection{Enzymes are powerful and highly Specific catalysts}

There are proteins for which ligand binding is simply a necessary first step
in their function. This is the case for the large and very important class
of proteins called enzymes. Enzymes bind to one or more ligands, called substrates, 
and convert them into chemically modified products, doing this over and over again with amazing rapidity.
They speed up reactions, often by a factor of a
million or more, without themselves being changed - that is, enzymes act
as catalysts that permit cells to make or break covalent bonds at will.

Enzymes can be grouped into functional classes based on the chemical
reactions they catalyze. Each type of enzyme is highly specific,
catalyzing only a single type of reaction.

\subsection{Lysozyme illustrates how an enzyme works}

To explain how enzymes catalyze chemical reactions, we will use the
example of lysozyme - an enzyme that acts as a natural antibiotic in egg
white, saliva, tears, and other secretions. Lysozyme severs the polysaccharide 
chains that form the cell walls of bacteria.

The reaction catalyzed by lysozyme is a hydrolysis: the enzyme adds a
molecule of water to a single bond between two adjacent sugar groups in
the polysaccharide chain, thereby causing the bond to break.
For a colliding water molecule to break a bond linking two sugars, the
polysaccharide molecule has to be distorted into a particular shape—the
transition state - in which the atoms around the bond have an altered
geometry and electron distribution. To distort the molecule in this way
requires a large input of energy (energy barrier) from random molecular collisions.

This is where the enzyme comes in. Like all enzymes, lysozyme has a
special binding site on its surface, termed an active site, that cradles the
contours of its substrate molecule. Here the catalysis of the chemical reaction 
occurs. Because its substrate is a polymer, lysozyme’s active site is a
long groove that holds six linked sugars at the same time. As soon as the
polysaccharide binds to form an enzyme - substrate complex, the enzyme
cuts the polysaccharide by catalyzing the addition of a water molecule to
one of its sugar-sugar bonds.

The chemistry that underlies the binding of lysozyme to its substrate is
the same as that for antibody binding - the formation of multiple non-covalent bonds.

The overall chemical reaction, from the initial binding of the polysaccharide 
on the surface of the enzyme to the final release of the severed
chains, occurs many millions of times faster than it would in the absence
of enzyme.

In reactions involving two or more
substrates, the active site also acts like a template or mold that brings
the reactants together in the proper orientation for chemistry to occur between them.
Binding to the enzyme also changes the shapes of substrates, 
bending bonds so as to drive a substrate toward a particular
transition state. Subsequent
steps in the reaction restore the side chain to its original state, so that the
enzyme remains unchanged after the reaction and can go on to catalyze
many more reactions.

\subsection{Most drugs inhibit enzymes}

Many of the drugs we take to treat or prevent illness work by blocking
the activity of a particular enzyme.


\subsection{Tightly Bound Small Molecules add extra functions to proteins}

Proteins often employ small nonprotein molecules to perform functions that would be
difficult or impossible using amino acids alone. When these small molecules are attached covalently and permanently
to their protein, they become an integral part of the protein molecule
itself.

Enzymes, too, make use of nonprotein molecules: they frequently have
a small molecule or metal atom associated with their active site that
assists with their catalytic function. Carboxypeptidase, an enzyme that
cuts polypeptide chains, carries a tightly bound zinc ion in its active site.

\section{How proteins are controlled}

The activities of cellular proteins are regulated in
a coordinated fashion so that the cell can maintain itself in an optimal
state, generating only those molecules it requires to thrive under the current conditions. 
By coordinating when - and how vigorously - proteins
function, the cell ensures that it does not deplete its energy reserves by
accumulating molecules it does not need or waste its stockpiles of critical substrates.

The regulation of protein activity occurs at many levels. At one level, the
cell controls how many molecules of each enzyme it makes by regulating 
the expression of the gene that encodes that protein.
At another level, the cell controls enzymatic activities by confining s
ets of enzymes to particular subcellular compartments, often - but
not always - enclosed by distinct membranes.
But the most rapid and general process used to adjust reaction
rates operates at the level of the enzyme itself. Although proteins can be
switched on - or switched off - by a variety of mechanisms, as we see
next, all in some way cause the protein to alter its shape, and therefore
its function.

\subsection{The catalytic activities of enzymes are often regulated by other Molecules}

By their catalytic action, enzymes generate a complex web of metabolic pathways 
composed of chains of chemical reactions in which the product of
one enzyme becomes the substrate of the next.

The most common type of control occurs when a molecule other than a
substrate specifically binds to an enzyme at a special regulatory site outside 
of the active site, altering the rate at which the enzyme converts its
substrates to products. In feedback inhibition, an enzyme acting early in
a reaction pathway is inhibited by a late product of that pathway.

Feedback inhibition is a negative regulation: it prevents an enzyme from
acting. Enzymes can also be subject to positive regulation, in which the
enzyme’s activity is stimulated by a regulatory molecule rather than being
shut down.

\subsection{Allosteric enzymes have Binding Sites That influence one another}

The regulatory molecule often has a shape that is totally
different from the shape of the enzyme’s preferred substrate. Indeed,
when this form of regulation was discovered in the 1960s, it was termed
allostery (from the Greek allo, “other,” and stere, “solid” or “shape”). As
more was learned about feedback inhibition, researchers realized that
many enzymes must have at least two different binding sites on their
surface: the active site that recognizes the substrates and one or more
sites that recognize regulatory molecules. Furthermore, the substrate and
regulatory sites must somehow “communicate” in a way that allows the
catalytic events at the active site to be influenced by the binding of the
regulatory molecule at its separate site.
The interaction between sites that are located on separate regions of a
protein molecule is now known to depend on conformational changes
in the protein: binding at one of the sites causes a shift in the protein’s
structure from one folded shape to a slightly different folded shape.

Many - if not most - protein molecules are allosteric: they can adopt
two or more slightly different conformations, and by a shift from one to
another, their activity can be regulated.

\subsection{Phosphorylation can control protein activity by Triggering a conformational change}

Another method commonly used by eucaryotic cells to regulate protein 
activity involves attaching a phosphate group covalently to one of
its amino acid side chains. Because each phosphate group carries two
negative charges, the enzyme-catalyzed addition of a phosphate group
to a protein can cause a major conformational change.
Removal of the phosphate group by a second enzyme returns the protein to its original
conformation and restores its initial activity.

This reversible protein phosphorylation controls the activity of many
different types of proteins in eucaryotic cells.
The addition and removal of phosphate groups from specific proteins
often occurs in response to signals that specify some change in a cell’s
state. And many of the signals generated by hormones and neurotransmitters are
carried from the plasma membrane to the nucleus by a cascade of protein
phosphorylation events.

Protein phosphorylation involves the enzyme-catalyzed transfer of the
terminal phosphate group of ATP to the hydroxyl group on a serine, threonine, 
or tyrosine side chain of the protein. This reaction is catalyzed by a
protein kinase. The reverse reaction - removal of the phosphate group,
or dephosphorylation - is catalyzed by a protein phosphatase.

For many proteins, a phosphate group is added to a particular side
chain and then removed in a continuous cycle. Phosphorylation cycles
of this kind allow proteins to switch rapidly from one state to another.

\subsection{GTP-Binding proteins are also regulated by the cyclic Gain and loss of a phosphate Group}

Eucaryotic cells have a second way to regulate protein activity by phosphate 
addition and removal. In this case, instead of being enzymatically
transferred from ATP to the protein, the phosphate is part of a guanine
nucleotide - either guanosine triphosphate (GTP) or guanosine diphosphate 
(GDP) - that is bound tightly to the protein. Such GTP-binding
proteins are in their active conformations with GTP bound; the protein
itself then hydrolyzes this GTP to GDP-releasing a phosphate - and flips
to an inactive conformation. As with protein phosphorylation, this process is reversible.

The GTP-binding proteins bind to other proteins to
control their activities, and their crucial role in intracellular signaling
pathways will be discussed in detail later.

\subsection{Nucleotide hydrolysis allows Motor proteins to produce large Movements in cells}

Conformational changes also play another important role in the operation of the cell: they enable
proteins whose major function is to move other molecules, the motor
proteins, to generate the forces responsible for muscle contraction and
many of the dramatic movements of cells.

How are shape changes in proteins used to generate orderly movements
in cells?

To make the series of conformational changes unidirectional - and force
the entire cycle to proceed in one direction - it is enough to make any
one of the steps irreversible. For most proteins that are able to walk in
a single direction for long distances, this irreversibility is achieved by
coupling one of the conformational changes to the hydrolysis of an ATP
molecule bound to the protein

Many motor proteins generate directional movement in this general way,
including the muscle motor protein myosin - which “runs” along actin
filaments to generate muscle contraction - and
the kinesin protein involved in chromosome movements at mitosis.

\subsection{Proteins often form large complexes That function as protein Machines}

It is clear that each central process in
a cell - such as DNA replication, protein synthesis, vesicle budding, and
transmembrane signaling - is catalyzed by a highly coordinated, linked
set of 10 or more proteins. In most such protein machines the hydrolysis
of bound nucleoside triphosphates (ATP or GTP) drives an ordered series
of conformational changes in some of the individual protein subunits,
enabling the ensemble of proteins to move coordinately.

\subsection{Covalent Modification controls the location and assembly of protein Machines}

It has recently become clear that
most protein machines form at specific sites in the cell and are activated
only when and where they are needed. This mobilization is generally
accomplished by the covalent addition of a modifying group to one or
more specific amino acid side chains on the participating proteins.

The set of covalent modifications that a protein contains at any moment
constitutes an important combinatorial regulatory protein code. The
attachment or removal of these modifying groups controls the behavior
of a protein, changing its activity or stability, its binding partners, or its
location inside the cell.

\section{Essential concepts}

\begin{itemize}
\item Living cells contain an enormously diverse set of protein molecules,
each made as a linear chain of amino acids covalently linked
together.
\item Each type of protein has a unique amino acid sequence that determines 
both its three-dimensional shape and its biological activity.
\item The folded structure of a protein is stabilized by noncovalent interactions 
between different parts of the polypeptide chain.
\item Hydrogen bonds between neighboring regions of the polypeptide
backbone often give rise to regular folding patterns, known as $\alpha$ helices and $\beta$ sheets.
\item The structure of many proteins can be subdivided into smaller globular 
regions of compact three-dimensional structure, known as protein
domains.
\item The biological function of a protein depends on the detailed chemical
properties of its surface and how it binds to other molecules, called
ligands.
\item When a protein catalyzes the formation or breakage of covalent
bonds in a ligand, the protein is called an enzyme and the ligand is
called a substrate.
\item At the active site of an enzyme, the amino acid side chains of the
folded protein are precisely positioned so that they favor the formation 
of the high-energy transition states that the substrates must
pass through to be converted to product.
\item The three-dimensional structure of many proteins has evolved so
that the binding of a small ligand can induce a significant change in
protein shape.
\item Most enzymes are allosteric proteins that can exist in two conformations 
that differ in catalytic activity, and the enzyme can be turned
on or off by ligands that bind to a distinct regulatory site to stabilize
either the active or the inactive conformation.
\item The activities of most enzymes within the cell are strictly regulated.
One of the most common forms of regulation is feedback inhibition,
in which an enzyme early in a metabolic pathway is inhibited by its
binding to one of the pathway’s end products.
\item Many thousands of proteins in a typical eucaryotic cell are regulated
either by cycles of phosphorylation and dephosphorylation, or by the
binding and hydrolysis of GTP by a GTP-binding protein.
\item The hydrolysis of ATP to ADP by motor proteins produces directed
movements in the cell.
\item Highly efficient protein machines are formed by assemblies of allosteric 
proteins in which conformational changes are coordinated to
perform complex cellular functions.
\item A regulatory protein code based on the covalent modification of multiple 
amino acid side chains allows each cell to control the location
and assembly of its protein complexes.
\item Starting from crude cell homogenates, individual proteins can be
obtained in pure form by using a series of chromatography steps.
Purification allows the detailed properties of a protein to be revealed
by biochemical techniques and its exact three-dimensional structure
to be determined.
\end{itemize}

\chapter{DNA and Chromosomes}


\chapter{DNA Replication, Repair and Recombination}